language/move-binary-format/src/control_flow_graph.rs

```diff
@@ -46,12 +50,21 @@ struct BasicBlock {
 pub struct VMControlFlowGraph {
     /// The basic blocks
     blocks: Map<BlockId, BasicBlock>,
+    /// Basic block ordering for traversal
+    traversal_successors: Map<BlockId, BlockId>,
+    /// Map from blocks with a loop's last continue to that loop's starts
```

Comment:
How is "last continue" defined (looking for a precise definition in terms of nodes and edges in the CFG)?